VC/Services/UserService.cs

```diff
@@ -44,19 +44,26 @@ public async Task<User> CreateUserAsync(UserCreateRequestDTO user)
             return _mapper.Map<User>(appUser);
         }
 
-        public Task<bool> DeleteUserAsync(string id)
+        public async Task<bool> DeleteUserAsync(string id)
         {
-            throw new NotImplementedException();
+
+            var user = await _userManager.FindByIdAsync(id);
+            if (user == null) { return false; }
+            var appUser = _mapper.Map<User>(user);
+            await _userManager.DeleteAsync(user);
```

Comment:
_userManager.DeleteAsync returns an IdentityResult, add an if statement to check for success